file/src/main/scala/akka/stream/alpakka/file/internal/ChangeableFileSink.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.file.internal
+
+import java.nio.file.StandardOpenOption
+
+import akka.stream.ActorAttributes.Dispatcher
+import akka.stream._
+import akka.stream.alpakka.file.internal.ChangeableFileSubscriber.ChangeableFileSubscriberMessages
+import akka.stream.impl.SinkModule
+import akka.stream.impl.Stages.DefaultAttributes.IODispatcher
+import akka.stream.impl.StreamLayout.Module
+
+import scala.concurrent.{Future, Promise}
+
+private[akka] final class ChangeableFileSink(options: Set[StandardOpenOption],
+                                             val attributes: Attributes,
+                                             shape: SinkShape[ChangeableFileSubscriberMessages])
+    extends SinkModule[ChangeableFileSubscriberMessages, Future[IOResult]](shape) {
```

Comment:
`SinkModule` is internal in Akka and not good to extend, use the public API `GraphStage`s instead